modules/core/src/core/lib/layer.js

```diff
@@ -362,6 +362,71 @@ export default class Layer {
     }
   }
 
+  // This method overwrites the current buffer and is used for multi-picking
+  _clearInstancePickingColor(color) {
+    const {instancePickingColors} = this.getAttributeManager().attributes;
+    const {state: attribute} = instancePickingColors;
+    const {value, size} = attribute;
+
+    const i = this.decodePickingColor(color);
+    value[i * size + 0] = 0;
+    value[i * size + 1] = 0;
+    value[i * size + 2] = 0;
+
+    // TODO: Optimize this to use sub-buffer update!
+    const models = this.getModels();
+    if (models) {
+      models.forEach(model => model.setAttributes({instancePickingColors: attribute}));
+    }
+  }
+
+  // This method overwrites the current buffer and is used for multi-picking
```

Comment:
Better comment: `// Sets all occurrences of the specified picking color to null picking color. Used for multi picking"`?